code/index.html

```diff
@@ -2,21 +2,73 @@
-
-    <section class="big-news">
-      <!-- The big news content -->
+    <!--Big news section-->
+    <section class="big-news-container">
+      <div class="news-card-image"><img src="./images/messi.jpg" alt="Lionel Messi"/></div>
+      <div class="news-card-text">
+        <h2>The end of Messi in Barcelona?</h2>
+      <p>Following FC Barcelonas huge 8-2 loss in the Champions League against Bayern Münich rumours are sparking about Lionel Messi wanting to leave the club. 
+        The 33-year old forward came to Barcelona only as a 13-year old and is by many considered to be the best football player the world has ever seen. 
+        He has won the <em>Champions League</em> four times, <em>La Liga</em> 10 times and <em>Copa del Rey</em> with Argentina as much as six times. 
+        He is also the only player to have won the Ballon dór six times, followed by his biggest rival Cristiano Ronaldo at five. 
+        The prize is awarded the worlds best footballer each year. 
+        A reason to Messis likely departure might be that Barcelonas new coach Ronald Koeman told Messi's close friend and striking partner, 
+        Luis Suarez, he does not wish the Uruguayan to stay at the club. 
+        During the last 12 months there has also been an open conflict between Messi and the club president Josep Maria Bartomeu.
+        According to sources, Messi has been unhappy with how the club is being run under Bartomeu. At the moment it looks like PSG and Man City are closest to signing the argentinian wizard.</p>
+        <h3>Messi's outstanding stats in FC Barcelona:</h3>
+        <ul>
+          <li>731 matches</li>
+          <li>634 goals</li>
+          <li>285 assist</li>
+        </ul>
+      </div>
     </section>
-
-    <section class="">
-      <!-- The other news  -->
+    <!--Other news section-->
+    <section class="other-news-container">
+      <div class="news-child"><img src="./images/koeman.jpg" alt="Ronald Koeman"/><p>New Barcelona coach Ronald Koeman</p></div>
+      <div class="news-child"><img src="./images/cristiano.jpg" alt="Cristiano Ronaldo"/><p>Messi's biggest rival Cristiano Ronaldo</p></div>
+      <div class="news-child"><img src="./images/family.jpg" alt="The Messi family"/><p>Is the Messi family looking for a new home?</p></div>
+      <div class="news-child"><img src="./images/messi-city.jpg" alt="Messi in Man City?"/><p>Could we be seeing Messi in Man City?</p></div>
+      <div class="news-child"><img src="./images/pepaguero.jpg" alt="Reason to move?"/><p>This could be a reason to move to City</p></div>
+      <div class="news-child"><img src="./images/messiaguero.jpg" alt="Good friends"/><p>Messi and Agüero are close friends</p></div>
+      <div class="news-child"><img src="./images/messi-dad.jpg" alt="Messi's father"/><p>Messi's father and agent Jorge Messi</p></div>
+      <div class="news-child"><img src="./images/messiballon.jpg" alt="Ballon dór"/><p>The one and only six time winner of Ballon dór</p></div>
```

Comment:
These could use some indentation and structure however 🙃 But good use of alt in the image tags!!